clients/client-auto-scaling/src/protocols/Aws_query.ts

```diff
@@ -10652,8 +10652,8 @@ const buildFormUrlencodedString = (formEntries: Record<string, string>): string
     .join("&");
 
 const loadQueryErrorCode = (output: __HttpResponse, data: any): string | undefined => {
-  if (data.Error.Code !== undefined) {
-    return data.Error.Code;
+  if (data.Error?.Code !== undefined) {
+    return data.Error?.Code;
```

Comment:
The optional chaining is only needed inside if checks, and not in return statements.
```suggestion
    return data.Error.Code;
```